perf_benchmark.py

```diff
@@ -43,9 +43,14 @@
     if func_type[0] == "hypothesis":
         # read path from arguments
         path = configuration.get_schema_path(tada_configuration_dict)
+    level = tada_configuration_dict.get("level")
```

Comment:
It would be more consistent with the previous arguments if `level` is being accessed through a function in `configuration`